### PR TITLE
[CAZ-1262] Update dashboard page and error messages

### PR DESCRIPTION
**Pull request by likeuwill**
## Motivation and Context

[JIRA](https://eaflood.atlassian.net/browse/CAZ-1262)
Manage users (only super admin)  that directs to manage-users page
Payment history (only super admin) directs to payment-history page
Upload your fleet (super and fleet admin) directs to csv-upload page
Make a payment (super and fleet admin) 
Create a direct debit mandate (super and fleet admin) directs to select-direct-debit page
admin dashboard
admin_password
user dashboard
user_dashboard

[CAZ-1219](https://eaflood.atlassian.net/browse/CAZ-1219)
tba_error

## Checklist:

- [x] It contains only changes required by issue (does not contain other PR)
- [x] Includes link to a issue (if apply)
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: app/views/organisations/new_name.html.haml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       - if @error
         = render 'common/error_summary',
           error_msg_div: '#govuk-error-company_name',
-          error_msg: 'Enter the company name'
+          error_msg: '[TBA] Enter the company name'
```
Comment (miqs1992) [inline app/views/organisations/new_name.html.haml]: Maybe move it to locales?

File: config/locales/en.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 en:
   form:
     errors:
-      missing: 'is required [TBA]'
-      maximum_length: 'is too long (maximum is %{length} characters) [TBA]'
+      missing: 'is required'
```
Comment (miqs1992) [inline config/locales/en.yml]: `'[TBA] %{attribute} is required` maybe this will work?

File: app/views/dashboard/index.html.haml [hunk 3]
```diff
@@ -5,40 +5,18 @@
...
+      = render 'payment_history'
+      = render 'upload'
+    .govuk-grid-row
+      = render 'make_payment'
+      = render 'select_direct_debit'
+  - else
+    .govuk-grid-row
+      = render 'upload'
+      = render 'make_payment'
+      = render 'select_direct_debit'
```
Comment (miqs1992) [inline app/views/dashboard/index.html.haml]: 👍

File: config/locales/en.yml (hunk 2 above)
Comment (likeuwill) [reply config/locales/en.yml]: First what I tried, doesn't work it's adding all message after attribute

**Pull request merged**